src/com/lftechnology/java/training/sanish/game/LeagueResult.java

```diff
@@ -0,0 +1,55 @@
+package com.lftechnology.java.training.sanish.game;
+
+import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Provide football league game results funcitons
+ *
+ * @autor Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class LeagueResult {
+    private static final Logger LOGGER = Logger.getLogger(LeagueResult.class.getName());
+
+    private LeagueResult() {
+    }
+
+    /**
+     * Display League game result of specific team
+     *
+     * @param games    {@link Game} list of games
+     * @param teamName {@link String} team name
+     * @author Sanish Maharjan <sanishmaharjan@lftechology.com>
+     */
+    public static void displayLeagueGameResultOfTeam(ArrayList<Game> games, String teamName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The type of the "games" object should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)